### ISSUE TITLE
Change peer to peer-info

### ISSUE THREAD

**Issue by geoah** [nimona/go-nimona-kad-dht#6]
I'm pretty sure that using the same peer structure from net will bite us in the ass eventually.
How can we differentiate peer from the DHT peer info but still share/transport peers between net/dht?

**Comment by superdecimal** [comment]
The peer structure from net is this one.
```
type Peer struct {
	ID        string   `json:"id"`
	Addresses []string `json:"addresses"`
	entity    *openpgp.Entity
}
```
What do you think will be the problem?

**Comment by geoah** [comment]
This has been resolved as a side-effect by #10
